### ISSUE TITLE
TileLayer rendering Western Hemisphere only

### ISSUE THREAD

**Issue by kafonek** [jupyter-widgets/ipyleaflet#311]
We have a mirror of [GIS Basemaps](https://services.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer) on our corporate network.  When I try to use the MapServer from arcgisonline.com, it renders the world as expected.  When I use the internal mirror, it renders North/South America side-by-side twice instead of the entire world.  Using the same tile layer with folium does render the entire world as expected.  I am not sure what differences there are between our internal mirror and the publicly available MapServer.

The code I'm using looks like this - 

```python
url = 'https://internal.domain/arcgis/rest/services/Basemaps/World_Streetmap/MapServer/tile/{z}/{y}/{x}'

m1 = folium.Map(tiles=url, attr='gis', zoom_start=0) # renders the world as expected

layer = ipyleaflet.basemap_to_tiles({'url' : url, 'attribution' : 'gis'})
m2 = ipyleaflet.Map(zoom=0)
m2.add_layer(layer) # renders just the western hemisphere (twice)
```

GIS/basemaps isn't my strong suit, have you seen this kind of behavior before?  Is it related to https://github.com/jupyter-widgets/ipyleaflet/issues/255 (which is WMSLayer not TileLayer..) or should I be looking for different debug?  Thanks in advance.

**Comment by martinRenou** [comment]
Thanks for the report! Can you share a screenshot?
I never have seen this kind of behavior, but I always used the default `EPSG3857` crs. If you know what is the difference (if there is one) between your mirror and the MapServer that would help a lot I think.
It's not obvious to me what folium does differently. I will try to compare the JavaScript code generated by folium and the JavaScript code that gets executed in ipyleaflet.

**Comment by martinRenou** [comment]
As discussed on gitter, it appears to be a `crs` issue. We need to apply the fix proposed in #255 and add something equivalent to the `Map` class
